test/Driver/response-file.swift

```diff
@@ -2,10 +2,28 @@
 // RUN: %swiftc_driver @%t.0.resp %s 2>&1 | %FileCheck %s -check-prefix=SHORT
```

Comment:
Please use `%target-build-swift -typecheck` instead of `%swiftc_driver` for these, so that they do less work and also compile for the correct platform.